Content.Shared/_Sunrise/Weapons/Ranged/ConsumableAmmoComponent.cs

```diff
@@ -0,0 +1,40 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._Sunrise.Weapons.Ranged
+{
+    [RegisterComponent, AutoGenerateComponentState, NetworkedComponent]
+    public sealed partial class ConsumableAmmoComponent : Component
+    {
+        [AutoNetworkedField]
+        [DataField("currentCharges")]
+        public int CurrentCharges = 0; 
+
+        [DataField("maxCharges")] 
+        public int MaxCharges = 30; 
+
+        // обязательно
+        [DataField("loadableItemIds", required: true)]
+        public List<string> LoadableItemIds = new() {};
```

Comment:
```suggestion
        public List<EntProtoId> LoadableItemIds = [];
```

Во-первых айди всегда лучше через EntProtoId указывать, во вторых листы определять через [], так код выглядит аккуратнее